Lib/test/support/import_helper.py

```diff
@@ -274,3 +274,27 @@ def mock_register_at_fork(func):
     # memory.
     from unittest import mock
     return mock.patch('os.register_at_fork', create=True)(func)
+
+
+@contextlib.contextmanager
+def ready_to_import(name=None, source=""):
+    # To fix import cycles:
+    from test.support import script_helper
+
+    # sets up a temporary directory and removes it
+    # creates the module file
+    # temporarily clears the module from sys.modules (if any)
+    # reverts or removes the module when cleaning up
```

Comment:
I personally find this a little hard to read. Is it a list of steps being followed? With no punctuation or capital letters I can't tell where something starts and another things begins.

Reply:
This is not my code, I just moved it :)
But, I can totally make it better, since I touch it anyway 👍

Reply:
Thanks!